src/components/settingFlyout/settingFlyout.js

```diff
@@ -23,6 +23,8 @@ const currSimulationLabel = {
   false: lang.STOPPED
 };
 
+const version = '1.0.0-preview.3';
```

Comment:
I think in the future it'd be better to read this from a file on disk.  That & we should think about whether we list the container tags in it instead of the top level repo tag.  For not it looks fine.

Reply:
we have config.js that I think we could reuse

Reply:
Yep, its on the todo. The config would be an easy place to put it.